packages/sampler/stories/qa/IncrementSlider.js

```diff
@@ -11,4 +42,13 @@ storiesOf('IncrementSlider', module)
 				<IncrementSliderDelayValue />
 			</div>
 		)
+	)
+	.add(
+		'spotlight behavior while dragging',
+		() => (
+			<div>
+				While holding down the knob (dragging), move the cursor quickly between knob and SliderButtons.
```

Comment:
Should this message indicate what should happen.  I suppose that either the test case has the instructions and the result or the sample should have both.  It doesn't make as much sense to have only half the info in two places.